workshops/Image Sharing Site/README.md

```diff
@@ -0,0 +1,459 @@
+---
+name: "Simple Image Sharing Site"
+description: "Build an image hosting site in 30 minutes"
+author: "@bezlin6mechminerz"
+---
+
+In this workshop, we will see how to make a simple site in which we can post images and view images posted by others. It will be super fun as you can host it and share your website with your friends and create a large collection of great pictures.
+
+The workshop will look like this.
+
+![Output video](https://cloud-75rs5g7wv.vercel.app/0ezgif.com-video-to-gif.gif)
+
+View a [live demo](https://simplesocialmedia.bezlin.repl.co)
+
+View the [final code](https://repl.it/@bezlin/simplesocialmedia#index.html)
+
+This workshop will take about 30 minutes.
```

Comment:
The rules state that the workshop should take around 20 minutes, so it will need to be shortened a little bit. It seems setting up Firebase takes a lot of time—maybe you can look into alternative image hosting providers that are easier to set up?